packages/@internationalized/number/src/NumberParser.ts

```diff
@@ -108,6 +108,19 @@ class NumberParserImpl {
 
   constructor(locale: string, options: Intl.NumberFormatOptions = {}) {
     this.locale = locale;
+    // see https://tc39.es/ecma402/#sec-setnfdigitoptions, when using roundingIncrement, the maximumFractionDigits and minimumFractionDigits must be equal
```

Comment:
note, this previously threw in Chrome and Node 22
`new Intl.NumberFormat('en-US', {roundingIncrement: 2})`